wger/settings_global.py

```diff
@@ -130,13 +136,15 @@
 
 AUTHENTICATION_BACKENDS = (
     'django.contrib.auth.backends.ModelBackend',
-    'wger.utils.helpers.EmailAuthBackend'
+    'wger.utils.helpers.EmailAuthBackend',
+    # 'guardian.backends.ObjectPermissionBackend',
```

Comment:
Remember to remove the commented code on the next commit.

Reply:
Thanks for the reminder. 👍🏽